crate/server/src/tests/cover_crypt_tests/integration_tests.rs

```diff
@@ -21,19 +16,16 @@ use cosmian_kms_utils::crypto::{
     generic::kmip_requests::{build_decryption_request, build_encryption_request},
 };
 
-use crate::test_utils;
+#[cfg(test)]
```

Comment:
I think this annotation is not needed as we are already in `tests` folder

Reply:
Yep. Txs